fairworkflows/fairstep.py

```diff
@@ -494,16 +494,26 @@ def _extract_outputs_from_function(func, additional_params) -> List[FairVariable
                          'FAIR step functions MUST have type hinting, '
                          'see https://docs.python.org/3/library/typing.html')
     if _is_generic_tuple(return_annotation):
+        return_sem_types = additional_params.get('out')
```

Comment:
Some alternatives for 'out':
1. 'return_types'
2. 'returns'
3. 'should_return'
I have a preference for 1 (I think it is better than 'out')

Reply:
Hmm, yes, these are all good. I think I have a preference for 2., since it is shorter and looks more like ```return```. Do you object if I use that one instead?

Reply:
That is fine, it might be slightly confusing because users might miss that it is means 'returns type' instead of 'returns value'. But I like that it is shorter too.

Reply:
So go ahead!